deliverables/README.md

```diff
@@ -2,31 +2,58 @@
 ###Intro to the Back End
 ####Required
 What is the difference between front- and back-end development?<br>
-Why do you need to install Ruby but not HTML/CSS?
+frontend is concerned with the user experience and look of the app, backend handles the business logic and database interactions<br>
+Why do you need to install Ruby but not HTML/CSS?  <br>
+ruby is the language that the server uses for logic, which outputs data displayed by a browser in html/css <br>
+since html and css are interpreted by the browser itself, its not something one installs <br>
```

Comment:
Correct.  Technically you do install it though - when you install your browser.  So the only difference is you need to install ruby only because it's not *already* installed.